zuul.d/ansible-navigator-jobs.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- job:
+    name: ansible-navigator-tox-smoke
+    parent: tox
```

Comment:
it is possible you want to use ansible-tox-py38 here, over tox

Reply:
this will allow you to use bindep.txt files for installs

Reply:
Is there anything special I have to do for that? Just switch it here and then drop a bindep.txt in the root of the other repo?

Reply:
Opened #849 to change it, but actually if I'm reading right, `tox` should be enough? https://opendev.org/zuul/zuul-jobs/src/branch/master/zuul.d/python-jobs.yaml#L36